schwab/streaming.py

```diff
@@ -866,7 +867,7 @@ class LevelOneOptionFields(_BaseFieldEnum):
         #: Expiration year
         EXPIRATION_YEAR = 12
 
-        #: MULTIPLIER
```

Comment:
Note: Comment changes like this are okay because they make things more consistent.

Reply:
Ok. Done.